Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -162,12 +162,12 @@ public class NavigationViewportDataSource: ViewportDataSource {
         let followingCameraOptions = options.followingCameraOptions
         
         if let location = rawLocation ?? passiveLocation {
-            if followingCameraOptions.centerUpdatesAllowed {
+            if followingCameraOptions.centerUpdatesAllowed || followingMobileCamera.center == nil {
                 followingMobileCamera.center = location.coordinate
                 followingCarPlayCamera.center = location.coordinate
             }
             
-            if followingCameraOptions.zoomUpdatesAllowed {
+            if followingCameraOptions.zoomUpdatesAllowed || followingMobileCamera.zoom == nil {
```

Comment:
Let's add a test case that verifies that in case if we set `zoomUpdatesAllowed` (and any other camera related property) to `false` `followingCarPlayCamera`/`followingMobileCamera` has correct non-nil default value.

Reply:
Thanks for the suggestion, I'll add test case for this.